### PR TITLE
Issue-3605. Example of customer fetching

### PR DESCRIPTION
**Pull request by kidinov**
Following the [feedback](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1903#issuecomment-788913802) I got, I've decided to split the "example" part into 3 subtasks. 1 PR per each functionality:
* Fetching by id (the current one)
* Fetching a list
* Creation

It's still just 1 commit here because it was already like that when I got the feedback =(

Part number 7 of [#3605](https://github.com/woocommerce/woocommerce-android/issues/3605), contains an example of the usage of FluxC functionality of fetching a single customer by id.

**Testing:**
1. Open the example
2. Sign in
3. Click "WOO" button
4. Scroll down and click "CUSTOMERS" button
5. Select site
6. Click "print customer"
7. Look into the logs

### CODE REVIEW

**Review by 0nko — CHANGES_REQUESTED** [review]
I forgot to mention this in the other PRs -- you need to add a migration script to `WellSqlConfig` that will create the table in an existing database. Otherwise you'll get an exception:

```
    --------- beginning of crash
2021-03-04 11:22:00.279 9123-9123/org.wordpress.android.fluxc.example E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android.fluxc.example, PID: 9123
    android.database.sqlite.SQLiteException: no such table: WCCustomerModel (code 1 SQLITE_ERROR): , while compiling: SELECT * FROM WCCustomerModel WHERE ((_id = ? OR (REMOTE_CUSTOMER_ID = ? AND LOCAL_SITE_ID = ?)))
        at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
        at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1045)
        at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:652)
        at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:590)
        at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:61)
        at android.database.sqlite.SQLiteQuery.<init>(SQLiteQuery.java:37)
        at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:46)
        at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1545)
        at android.database.sqlite.SQLiteQueryBuilder.query(SQLiteQueryBuilder.java:592)
        at android.database.sqlite.SQLiteQueryBuilder.query(SQLiteQueryBuilder.java:502)
        at com.yarolegovich.wellsql.SelectQuery.execute(SelectQuery.java:208)
        at com.yarolegovich.wellsql.SelectQuery.getAsModel(SelectQuery.java:155)
        at com.yarolegovich.wellsql.SelectQuery.getAsModel(SelectQuery.java:151)
        at org.wordpress.android.fluxc.persistence.CustomerSqlUtils.insertOrUpdateCustomer(CustomerSqlUtils.kt:40)
        at org.wordpress.android.fluxc.store.WCCustomerStore$fetchSingleCustomer$2.invokeSuspend(WCCustomerStore.kt:46)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
        ```

**Comment by kidinov** [comment]
@0nko indeed. Fixed [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1905/commits/b3d39b80b3391db390b01a8d51d82188489ae19f) + [conflict resolved](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1905/commits/2c44f66781a555197f6959088b127501a8a63d8b)

File: example/src/main/java/org/wordpress/android/fluxc/example/ui/common/FragmentExt.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package org.wordpress.android.fluxc.example.ui.common
+
+import androidx.fragment.app.Fragment
+import org.wordpress.android.fluxc.example.SiteSelectorDialog
+import org.wordpress.android.fluxc.example.ui.StoreSelectorDialog
+
+fun Fragment.showStoreSelectorDialog(selectedPos: Int, listener: StoreSelectorDialog.Listener) {
```
Comment (0nko) [inline example/src/main/java/org/wordpress/android/fluxc/example/ui/common/FragmentExt.kt:7]: Nice 👏

**Review by 0nko — APPROVED** [review]
Looking great! 🥇

**Pull request merged**